worlds/smz3/__init__.py

```diff
@@ -479,20 +479,27 @@ def write_spoiler(self, spoiler_handle: TextIO):
             self.world.spoiler.unreachables.update(self.unreachable)
 
     def JunkFillGT(self):
+        poolLength = len(self.world.itempool)
+        junkPool = [(i, self.world.itempool[i]) for i in range(0, poolLength) 
+                    if self.world.itempool[i].classification in (ItemClassification.filler, ItemClassification.trap)]
```

Comment:
would be easier on the eyes to only collect the indices of candidates, and use `world.itempool[junk[i]]` below

Reply:
Done in https://github.com/ArchipelagoMW/Archipelago/pull/886